### ISSUE TITLE
Hide SSID

### ISSUE THREAD

**Issue by kris14an** [Coopdis/easyMesh#22]
Is it possible to hide SSID wifi network ?

**Comment by vkynchev** [comment]
No. As far as I know. But If you want you can have a look at [painlessMesh](https://gitlab.com/BlackEdder/painlessMesh). There you can set only one SSID for the whole mesh. Eliminating the need of separate node SSID...

**Comment by kris14an** [comment]
Thanks i'll try
